crates/task-impls/src/network.rs

```diff
@@ -446,4 +439,74 @@ impl<
 
         None
     }
+
+    /// handle `VidDisperseSend`
+    fn handle_vid_disperse_proposal(
+        &self,
+        vid_proposal: Proposal<TYPES, VidDisperse<TYPES>>,
+        sender: &<TYPES as NodeType>::SignatureKey,
+        maybe_action: Option<HotShotAction>,
```

Comment:
nit: I don't think we need this argument, we can just use `Some(HotShotAction::VidDisperse)` in function